packages/front-end/components/Features/AttributeModal.tsx

```diff
@@ -103,14 +106,33 @@ export default function AttributeModal({ close, attribute }: Props) {
         refreshOrganization();
       })}
     >
-      <Field label="Attribute" {...form.register("property")} />
+      <Field
+        label={
+          <>
+            Attribute{" "}
+            <Tooltip body={"This is the attribute name used in the SDK"} />
+          </>
+        }
+        required={true}
+        {...form.register("property")}
+      />
       {attribute && form.watch("property") !== attribute ? (
         <div className="alert alert-warning">
           Be careful changing the attribute name. Any existing targeting
           conditions that use this attribute will NOT be updated automatically
           and will still reference the old attribute name.
         </div>
       ) : null}
+      <div className="form-group">
+        <label>
+          Description <small className="text-muted">(optional)</small>
+        </label>
```

Comment:
Pass the `label` prop into Field instead.  That will hook it up properly for accessibility